projects/etos_suite_runner/src/etos_suite_runner/lib/suite.py

```diff
@@ -60,6 +61,21 @@ def __init__(self, etos: ETOS, environment: dict, main_suite_id: str) -> None:
         self.test_suite_started = {}
         self.test_suite_finished = {}
 
+    @property
+    def failed(self) -> bool:
+        """Property indicating whether the sub-suite has had failures."""
+        if self._failed_flag_set:
+            return self._failed
+        verdict = self.outcome().get("verdict")
+        conclusion = self.outcome().get("conclusion")
+        return "FAILED" in (verdict, conclusion)
```

Comment:
I don't think we should set `failed` if the verdict is failed. Verdict does not mean that the sub suite failed. It could be a genuine verdict. The verdict of sub suites should be evaluated in the results method in the Suite class. The result will be exactly the same, but it will allow us to do more in the future without editing up here.

Reply:
This change won't be needed if we proceed with the current proposal of exit code handling/verdict matching in the Test Runner:
https://github.com/eiffel-community/etos-test-runner/pull/49/files